sdk/communication/azure-communication-administration/tests/identity/recordings/test_communication_identity_client.test_create_user.yaml

```diff
@@ -0,0 +1,41 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept:
+      - application/json
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '0'
+      Date:
+      - Thu, 08 Oct 2020 20:28:44 GMT
+      User-Agent:
+      - azsdk-python-communication-administration/1.0.0b2 Python/3.8.5 (Linux-5.4.0-48-generic-x86_64-with-glibc2.29)
+      x-ms-return-client-request-id:
+      - 'true'
+    method: POST
+    uri: https://communication27b5151c.communication.azure.com/identities?api-version=2020-07-20-preview2
```

Comment:
Dont you need to sanitize/redact these resource names?

Reply:
These resource are in only available in the context of test method. So no need I guess since It gets removed after the test